android/assets/jsons/translations/template.properties

```diff
@@ -41,6 +41,7 @@ Consumes 1 [resource] =
 Consumes [amount] [resource] = 
 Required tech: [requiredTech] = 
 Requires [PolicyOrNationalWonder] = 	
+Triggers a global alert upon completion = 
```

Comment:
Since the unique is defined in one of the Json files, we don't need to define it in the template file - only non-json, UI elements